tools.go

```diff
@@ -0,0 +1,6 @@
+// +build tools
+
+package main
+
+import _ "golang.org/x/lint"
+import _ "golang.org/x/tools/cmd/goimports"
```

Comment:
ref https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module